apps/web/drizzle/migrations/0029_normalize_is_bot.sql

```
@@ -0,0 +1,24 @@
+-- Wave 4a: Normalize click_events.is_bot to NOT NULL (online-safe)
+--
+-- Strategy: use CHECK NOT VALID + VALIDATE to avoid ACCESS EXCLUSIVE lock
+-- during the full table scan for NOT NULL enforcement.
+
+-- 1. Backfill existing NULLs to false
+UPDATE "click_events" SET "is_bot" = false WHERE "is_bot" IS NULL;--> statement-breakpoint
```

SQLFluff (4.1.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)